src/coreclr/vm/appdomain.cpp

```diff
@@ -3695,15 +3695,12 @@ PEAssembly * AppDomain::BindAssemblySpec(
         {
 
             {
-                // Use CoreClr's fusion alternative
-                CoreBindResult bindResult;
+                ReleaseHolder<BINDER_SPACE::Assembly> bindResult;
+                hrBindResult = pSpec->Bind(this, &bindResult);
 
-                pSpec->Bind(this, FALSE /* fThrowOnFileNotFound */, &bindResult);
-                hrBindResult = bindResult.GetHRBindResult();
-
-                if (bindResult.Found())
+                if (bindResult)
```

Comment:
Nit: I would rename the local variable to something like `boundAssembly` - basically include the word "Assembly" in it. It feels weird to read the code below and use a "result" as an assembly. (Not that it was better before, it was even worse)

Reply:
Right. I actually thought that `boundAssembly` could be a better name here. `pBinder` would be another choice. I am not a fan of 'p' prefixes, but it is used in many places in the binder code. 
Anyways, `boundAssembly`  feels better here.

I think I will make another pass over variables typed as `AssemblyBinder*`. We have a variety of [ load | context | binder ] combinations when naming these. Perhaps calling them `binder` or `pBinder`, in majority of cases at least, would make it more consistent.